README.md

```diff
@@ -1,2 +1,41 @@
 # U-Prep-Pathways-Platform
 This repository holds NiJeL's work with the University Preparatory Academy in Detroit, MI on the U Prep Pathways Platform
+
+## Getting Started
+To clone and run this application, you'll need [Git](https://git-scm.com), [PostgreSQL](https://www.postgresql.org/download/) and [Node.js](https://nodejs.org/en/download/) (which comes with [npm](http://npmjs.com)) installed on your computer. From your command line:
+
+```bash
+# Clone this repository
+$ git clone https://github.com/NiJeLorg/U-Prep-Pathways-Platform.git
+
+# Go into the repository
+$ cd electron-markdownify
```

Comment:
Also we are missing a section explaining how to setup the .env file with all the proper configurations